### PR TITLE
[bugfix] Allow expiry month and year to be null

### PR DESCRIPTION
**Pull request by zeulb**
Fixes
```
Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[1]’
```

Since `expiryMonth` and `expiryYear` are optional, we have to explicitly pass in `[NSNull null]` because `NSDictionary` doesn't allow nil assignment

https://stackoverflow.com/questions/13810875/inserting-nil-objects-into-an-nsdictionary

### CODE REVIEW

File: ios/RNCardscan.m [hunk 1]
```diff
@@ -35,8 +35,8 @@ - (void)userDidScanCard:(ScanViewController * _Nonnull)scanViewController credit
         @"action" : @"scanned",
         @"payload": @{
             @"number": number,
-            @"expiryMonth": expiryMonth,
-            @"expiryYear": expiryYear
+            @"expiryMonth": expiryMonth ?: [NSNull null],
+            @"expiryYear": expiryYear ?: [NSNull null]
```
Comment (jaimejiyepark) [inline ios/RNCardscan.m]: qq: Do we need to update these objc files if we change the swift protocols?

Comment (zeulb) [reply ios/RNCardscan.m]: Hmm, I'm not very familiar with Swift but you might have to update this bridge file if any if `number`, `expiryMonth`, or `expiryYear` changes although I don't see that happening a lot.

Comment (jaimejiyepark) [reply ios/RNCardscan.m]: hmmm i see. how do i update the file for example if we decide to break interface and change the variable name `number` to `last4`. would i have to do it manually?

Comment (zeulb) [reply ios/RNCardscan.m]: Yup, this method takes in the CreditCard object from Swift so you will have to update https://github.com/getbouncer/cardscan-ios-react/pull/3/files#diff-7a94af744d0384802abe0b7234d25ce3R30 above appropriately and this payload block is what we send back to React Native https://github.com/getbouncer/cardscan-ios-react/blob/master/example/App.js#L25

**Comment by jaimejiyepark** [comment]
lgtm #shipit

**Pull request merged**